src/Microsoft.AspNetCore.Razor.Design/project.json

```diff
@@ -12,7 +12,10 @@
       "taghelper",
       "aspnetcore",
       "taghelpers"
-    ]
+    ],
+    "files": {
+      "include": [ "buildCrossTargeting/*", "build/*" ]
```

Comment:
Instead of doing your own cross targeting, why not just tap into the existing cross targeting infrastructure? If you pack your targets in `build/{tfm}/*.targets` instead, they'll correctly invoke your target in the "inner build" instead of the outer build.